assets/src/css/frontend/donation-grid.scss

```diff
@@ -263,3 +259,125 @@
 .my-mfp-slide-bottom.mfp-removing.mfp-bg {
 	opacity: 0;
 }
+
+.white-popup {
```

Comment:
@Sidsector9 I feel these magnific styles should exist outside of the grid stylesheet within their own since likely these styles will be used in other places as well and finding them here seems difficult for developer who don't know. @kevinwhoffman what are your thoughts?

Reply:
I agree. Sid and I are going to meet and review ITCSS and I think reworking these class names and locations would be a good first task.